### PR TITLE
Format corrections of JWTRule document

### PR DESCRIPTION
**Pull request by sathishsw**
Subject: Apply relevant formatting at JWTRule page.
Description:
+ Corrected the command texts with backtick so that code block will be rendered.
+ Updated TODO with comment enclosure block.
+ Formatted hyperlink

### CODE REVIEW

File: security/v1beta1/jwt.pb.html [hunk 1]
```diff
@@ -184,7 +184,7 @@ <h2 id="JWTRule">JWTRule</h2>
     claim: nested.key.group
 </code></pre>
 <p>[Experimental] This feature is a experimental feature.
-TODO: Update the status whenever this feature is promoted</p>
+[#comment: TODO: Update the status whenever this feature is promoted.]</p>
```
Comment (howardjohn) [inline security/v1beta1/jwt.pb.html:187]: I am not sure what this syntax is? Whatever it is it doesn't seem to properly hide it?

**Review by howardjohn — APPROVED** [review]
lgtm aside from concern around the `#comment` thing

File: security/v1beta1/jwt.pb.html (hunk 1 above)
Comment (sathishsw) [reply security/v1beta1/jwt.pb.html]: Yes, agree! Moving the comment line to the hide format in the subsequent push.

**Review by ericvn — APPROVED** [review]

**Comment by ericvn** [comment]
@sathishsw You just need to satisfy the CLA.

**Pull request merged**